docs/concepts/services-networking/service.md

```diff
@@ -203,6 +203,35 @@ having working [readiness probes](/docs/tasks/configure-pod-container/configure-
 
 ![Services overview diagram for iptables proxy](/images/docs/services-iptables-overview.svg)
 
+### Proxy-mode: ipvs
```

Comment:
This should say `[alpha]` or have some sort of note in bold letters at the top that this is an alpha feature and not recommended for production clusters yet.

Reply:
+1